.travis.yml

```diff
@@ -50,12 +50,23 @@ branches:
 
 matrix:
   include:
+    - php: 7.1.8
+    - php: 7.1.9
+    - php: 7.1.11
+    - php: 7.1.15
+    - php: 7.1.17
+    - php: 7.1.20
+    - php: 7.1.24
     - php: 7.1
     - php: 7.1
       env: setup=lowest
     - php: 7.1
       env: setup=stable
 
+    - php: 7.2.0
+    - php: 7.2.3
+    - php: 7.2.5
+    - php: 7.2.8
```

Comment:
@kylekatarnls isn't it best to also add 7.2.12 which caused https://github.com/briannesbitt/Carbon/issues/1503 ?

Reply:
7.2.12 is currently 7.2 (last patch by default), so 7.2.12 does not need to be added until the next date change impacting 7.2.x. If we yet add it, we will run the tests twice on the same version. Then even if PHP released 7.2.13 tomorrow, if the date/time API did not change on this patch, we will not add this specific version to .travis.yml. The goal is to run at least once the tests on one PHP version between each DateTime API change.